superbench/config/azure/inference/standard_nc96ads_a100_v4.yaml

```diff
@@ -101,6 +101,8 @@ superbench:
     computation-communication-overlap:
       <<: *default_pytorch_mode
       enable: false
+    dist-inference-simulation:
```

Comment:
dist-inference

Reply:
Please remove all simulation

Reply:
Also, do we need to add NDv4 and NDmv4 only?

Reply:
Fixed